src/Electron/electron.ts

```diff
@@ -268,7 +294,7 @@ export class ElectronContainer extends WebContainerBase {
             main process we can just directly set the name.
         */
         electronWindow["name"] = (this.isRemote)
-            ? (<any>this.internalIpc).sendSync("setWindowName", { id: electronWindow.id, name: windowName })
+            ? (<any>this.internalIpc).sendSync("desktopJS.window-setname", { id: electronWindow.id, name: windowName })
```

Comment:
Renamed this channel for consistency with the other names and to make source clear.